examples/cloudrun/main.tf

```diff
@@ -24,14 +24,14 @@ provider "google-beta" {
 
 # [START cloudloadbalancing_ext_http_cloudrun]
 module "lb-http" {
-  source  = "GoogleCloudPlatform/lb-http/google//modules/serverless_negs"
-  version = "~> 6.2.0"
```

Comment:
So we have explicit versions here for surfacing in c.g.c. Is this giving an error otherwise?

Reply:
Hi @bharathkkb Yes, as 6.2.0 doesn't support lables.

Reply:
As discussed we'll update the example post-release that support labels. #262